Reactions/ReactorUtils.H

```diff
@@ -104,8 +104,8 @@ box_flatten(
     mass_frac[n] = rhoYn;
   }
   const amrex::Real rho_inv = 1.0 / rho;
-  for (int n = 0; n < NUM_SPECIES; n++) {
-    mass_frac[n] *= rho_inv;
+  for (amrex::Real& n : mass_frac) {
+    n *= rho_inv;
```

Comment:
this is stupid but since we use `n` to index things, can we use `mf` instead of `n` here? That way we know that we are *= a massfrac?

Reply:
...or Yn

Reply:
Yn is even better lol